dev/Expander/Expander_themeresources.xaml

```diff
@@ -109,6 +124,8 @@
     <HorizontalAlignment x:Key="ExpanderHeaderHorizontalContentAlignment">Stretch</HorizontalAlignment>
     <VerticalAlignment x:Key="ExpanderHeaderVerticalContentAlignment">Center</VerticalAlignment>
     <Thickness x:Key="ExpanderHeaderPadding">16,0,0,0</Thickness>
+    <Thickness x:Key="ExpanderHeaderPaddingExpandedUp">16,1,0,0</Thickness>
+    <Thickness x:Key="ExpanderHeaderPaddingExpandedDown">16,0,0,1</Thickness>
```

Comment:
@tashatitova @ranjeshj @StephenLPeters, instead of creating new resources I could do all the manipulations in the code behind based on the border thickness given to Expander.
Because the Divider line is in between the Header and Content they both need to lose Border Thickness on connecting side. That changes visual padding by the size of the stroke in the Header content.

Resource `ExpanderDividerMargin` exists to stop divider going out of the border of Expander (because expander has  backgrundSizing=innerBorderEdge):

Without `ExpanderDividerMargin`:
![image](https://user-images.githubusercontent.com/21356912/116955282-f0a58680-ac46-11eb-9582-b44ff14d6acc.png)
With:
![image](https://user-images.githubusercontent.com/21356912/116955289-f733fe00-ac46-11eb-9e90-e4152c2cedb3.png)